pandas/io/gbq.py

```diff
@@ -75,7 +75,7 @@ def read_gbq(query, project_id=None, index_col=None, col_order=None,
         see `BigQuery SQL Reference
         <https://cloud.google.com/bigquery/sql-reference/>`__
 
-    **kwargs : Arbitrary keyword arguments
+    `**kwargs` : Arbitrary keyword arguments
```

Comment:
Did you have a problem with this? (as the docstring is a raw string, it should work like that?)